contracts/contracts/coordination/Coordinator.sol

```diff
@@ -401,17 +407,26 @@ contract Coordinator is AccessControlDefaultAdminRules, FlatRateFeeModel {
         return getParticipantFromProvider(rituals[ritualId], provider);
     }
 
+    function isEncryptionAuthorized(
+        uint32 ritualId,
+        bytes memory evidence,
+        bytes memory ciphertextHeader
+    ) external view returns (bool) {
+        Ritual storage ritual = rituals[ritualId];
+        require(getRitualState(ritual) == RitualState.FINALIZED, "Ritual not finalized");
+        return ritual.accessController.isAuthorized(ritualId, evidence, ciphertextHeader);
```

Comment:
Thinking a bit more about this - @vzotova , @cygnusv are there any security risks to having the Coordinator contract call some random access controller contract? (cc @KPrasch)

Reply:
Not a problem since all the call chain is made of `view` methods.

Reply:
👍